cli/lib/kontena/cli/master/join_command.rb

```diff
@@ -9,9 +9,9 @@ class JoinCommand < Kontena::Command
 
     def execute
       params = []
-      params << "--join #{self.invite_code.shellescape}"
+      params += ["--join", self.invite_code.shellescape]
```

Comment:
No need to shellescape anymore with the array syntax.

(it was needed to add quotes in case a string had spaces)